### PR TITLE
Add new source: PhenixScans

### PR DESCRIPTION
**Pull request by MozkaGit**
## Checklist:
- [x] Updated source's version for individual source changes
- [x] Updated all sources' versions for template changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device

## Changes:
- Add new source [PhenixScans](https://phenixscans.fr/)

## PhenixScans Screenshots:

| Source Version| Browse | In-app Safari |
| :-: | :-: | :-: |
| source_version | source_browse | source_safari |

| Chapters list | Chapter pages | Search |
| :-: | :-: | :-: |
| source_chapters_list | source_chapter | source_search

**Thank you in advance for the review.**

### CODE REVIEW

**Comment by Skittyblock** [comment]
prs for different things should remain separate, I'm going to reopen your towerofelectrica pr and you should remove that change from this pr

File: src/rust/mangastream/sources/phenixscans/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+aidoku = { git = "https://github.com/Aidoku/aidoku-rs/" }
+mangastream_template = { path = "../../template" }
```
Comment (Skittyblock) [inline src/rust/mangastream/sources/phenixscans/Cargo.toml:12]: should be a newline
```suggestion
mangastream_template = { path = "../../template" }

```

File: src/rust/mangastream/sources/phenixscans/res/filters.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		"name": "Type",
+		"options": [
+			"All",
+			"Manga",
+			"Manhwa",
+			"Manhua",
+			"Comic"
+		]
+	}
+]
```
Comment (Skittyblock) [inline src/rust/mangastream/sources/phenixscans/res/filters.json:26]: newline 
```suggestion
]

```

File: src/rust/mangastream/sources/phenixscans/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+#[modify_image_request]
+fn modify_image_request(request: Request) {
+	get_instance().modify_image_request(request)
+}
+
+#[handle_url]
+pub fn handle_url(url: String) -> Result<DeepLink> {
+	get_instance().handle_url(url)
+}
```
Comment (Skittyblock) [inline src/rust/mangastream/sources/phenixscans/src/lib.rs:55]:
```suggestion
}

```

**Review by Skittyblock — CHANGES_REQUESTED** [review]

**Comment by MozkaGit** [comment]
Hello, the requested changes have been implemented.

**Review by Skittyblock — APPROVED** [review]

**Pull request merged**